src/processing/minify.rs

```diff
@@ -1,35 +1,40 @@
-use anyhow::anyhow;
 use minify_js::TopLevelMode;
 
 /// perform JS minification
-pub fn minify_js(bytes: &[u8], mode: TopLevelMode) -> anyhow::Result<Vec<u8>> {
+pub fn minify_js(bytes: Vec<u8>, mode: TopLevelMode) -> Vec<u8> {
     let mut result: Vec<u8> = vec![];
     let session = minify_js::Session::new();
-    minify_js::minify(&session, mode, bytes, &mut result)
-        .map_err(|err| anyhow!("Failed to minify JS: {err}"))?;
 
-    Ok(result)
+    match minify_js::minify(&session, mode, &bytes, &mut result) {
+        Ok(()) => result,
+        Err(err) => {
+            tracing::warn!("Failed to minify JS: {err}");
+            bytes
+        }
+    }
 }
 
 /// perform CSS minification
-pub fn minify_css(bytes: Vec<u8>) -> anyhow::Result<Vec<u8>> {
+pub fn minify_css(bytes: Vec<u8>) -> Vec<u8> {
     use css_minify::optimizations::*;
 
     if let Ok(css) = std::str::from_utf8(&bytes) {
-        Ok(Minifier::default()
-            .minify(css, Level::Three)
-            .map_err(|err| anyhow!("Failed to minify CSS: {err}"))?
-            .into_bytes())
-    } else {
-        Ok(bytes)
+        match Minifier::default().minify(css, Level::Three) {
```

Comment:
Kind of unrelated to his PR, but I wonder if we should change the default level to `Level::Two`, as the docs for `Level::Three` mention: "It is a danger optimizations, because ordering of your css code may be changed"

Reply:
I lowered it to `One`, as the docs say that even `Two` can be dangerous, and because `minify_html` has that as a default too.